flight/Modules/Stabilization/stabilization.c

```diff
@@ -831,6 +947,27 @@ static void stabilizationTask(void* parameters)
 					PIOS_Assert(0); /* Shouldn't happen, per above */
 					break;
 
+				case STABILIZATIONDESIRED_STABILIZATIONMODE_LQG:
+#if defined(PIOS_INCLUDE_LQG)
+					if (lqg[i] && lqg_is_solved(lqg[i])) {
+						if (reinit) {
+							lqg_set_x0(lqg[i], gyro_filtered[i]);
+						}
+						lqg_in_use = true;
+
+						/* Store to rate desired variable for storing to UAVO */
+						rateDesiredAxis[i] = bound_sym(raw_input[i], settings.ManualRate[i]);
+
+						/* Compute the inner loop */
+						actuatorDesiredAxis[i] = lqg_controller(lqg[i], gyro_filtered[i], rateDesiredAxis[i]);
+						/* The LQG controller bounds data already, but whatever. */
+						actuatorDesiredAxis[i] = bound_sym(actuatorDesiredAxis[i], 1.0f);
+						break;
+					}
+#endif
+					/* Fall through to rate, if LQR gains haven't been solved yet, or if
```

Comment:
nice

Reply:
We should make a fallthrough macro so we can start to get ahead of GCC8 etc?